lib/main.py

```diff
@@ -15,6 +15,8 @@
 
 BACKGROUND = None
 
+util.setGlobalProperty('running', '')
```

Comment:
I don't think we need this. Both because we'll be changing this to '1' anyways and because if it is already '1' here then we should probably either abort (because it's already running) or better (but much more complicated) try to recover the existing instance.

I think changing this to 'has.run' might be nice though.